raiden/tests/api/test_api.py

```diff
@@ -592,7 +623,9 @@ def test_break_blockchain_events(
     # the provided token_address
     api_test_context.specify_token_for_channelnew('0x61c808d82a3ac53231750dadc13c777b59310bd9')
     request = grequests.get(
-        'http://localhost:{port}/api/1/events/tokens/0x61c808d82a3ac53231750dadc13c777b59310bd9?from_block=5&to_block=20&token_address=0x167a9333bf582556f35bd4d16a7e80e191aa6476'.format(
+        'http://localhost:{port}/api/1/events/tokens/0x61c808d82a3ac53231750dadc13c777b59310bd9'
```

Comment:
Don't know how to provide an argument both times through `url_for()`. The test is to see if we can overwrite the arguments.